install/kubernetes/helm/subcharts/gateways/templates/preconfigured.yaml

```diff
@@ -173,13 +226,41 @@ metadata:
     release: {{ .Release.Name }}
 spec:
    workloadLabels:
-     istio: mcgateway
+     istio: ingressgateway
    filters:
    - listenerMatch:
        portNumber: 15443
        listenerType: GATEWAY
      filterName: envoy.filters.network.sni_cluster
      filterType: NETWORK
      filterConfig: {}
+   - listenerMatch:
+       portNumber: 15443
+       listenerType: GATEWAY
```

Comment:
is this a dup of line 231-233?

Reply:
or a different envoy filter but listen on the same port?

Reply:
different filter